scipy/spatial/tests/test_kdtree.py

```diff
@@ -216,47 +216,12 @@ def test_vectorized_query_multiple_neighbors(self):
         assert_(np.all(~np.isfinite(d[:, :, -s:])))
         assert_(np.all(i[:, :, -s:] == self.kdtree.n))
 
-    @pytest.mark.parametrize('r', [0.8, 1.1])
-    def test_single_query_all_neighbors(self, r):
+    def test_query_raises_for_k_none(self):
         np.random.seed(1234)
+        r = 0.8
         point = np.random.rand(self.kdtree.m)
```

Comment:
We don't need this to test that it raises. Just use the default with a float for `x`.

Reply:
You're right - thanks!